components/infobox/commons/infobox_league.lua

```diff
@@ -239,6 +239,13 @@ function League:createInfobox()
 					return
 				end
 
+				-- if the event is finished do not show the button
+				local osdateCuttoff = DateExt.parseIsoDate(endDate)
+				osdateCuttoff.day = osdateCuttoff.day + 2
```

Comment:
Why the +2?

Reply:
People are unlikely to look for accommodation on the last day, so I'm fine with cutting it on cutOff date

Reply:
what about 1-day events?